crypto/ledger/ledger_secp256k1.go

```diff
@@ -48,6 +47,10 @@ type (
 	}
 )
 
+func SetDiscoverLedger(fn discoverLedgerFn) {
```

Comment:
So I think the SDK will want to keep the `crypto/ledger/ledger_secp256k1.go` file as is and have app chains define their own custom functionality to support the desired application (eg: Ethereum)